app/controllers/SingleRecordController.scala

```diff
@@ -45,7 +46,7 @@ class SingleRecordController @Inject() (
   dataCleansingService: DataCleansingService,
   val controllerComponents: MessagesControllerComponents,
   view: SingleRecordView
-)(implicit ec: ExecutionContext)
+)(implicit @unused ec: ExecutionContext)
```

Comment:
can you just remove this if it is unused?

Reply:
yeah, something that I asked in the chat yesterday, we can't remove explicits - it is needed. But for some reasons, it shows the warning so we're suppressing it.